### DISCUSSION TITLE
why would a known TF and it's motif not show up in our results?

### DISCUSSION THREAD

**Discussion by sid5427** [aertslab/scenicplus#178] [Q&A]
Hi Seppe,

We are running scenic on a mouse dataset, and are interested in certain cell clusters. What we see is a certain TF showing no enrichment, i.e. it's motif is not enriched in the regions associated with it's target genes (We did restrict the search space to TADS). The TF of interest is Gfi1 and it's motif is certainly present in motifs-v10nr_clust-nr.mgi-m0.001-o0.0.tbl file. We know this TF is important and a good marker of activity for these clusters.

I would appreciate a discussion on when a certain motif is not deemed significant, and in turn, it's associated TF is not included in the downstream analysis, thus not reporting any TF-region-Gene interactions.

Thanks,
Sid

**Comment by ramadatta** [comment]
Hi, I have the same question as @sid5427 , you answer would be highly appreciated.

**Comment by SeppeDeWinter** [comment]
Hi both

This is a difficult question to answer without additional context ...
If you provide me more details and troubleshooting steps you have already tried (i.e. you can try to track where the TF of interest is lost), I might be able to help further.

All the best,

Seppe

**Comment by ramadatta** [comment]
Hi @SeppeDeWinter ,

Thank you for the comment.

I came across [this](https://github.com/aertslab/pySCENIC/issues/534) thread which threw some light on why some TF were not showing up in the end results.

Would like to hear your thoughts on this.

Best regards
Datta

**Comment by teng-gao** [comment]
Hi @SeppeDeWinter ,

I am also curious about this. Could you explain a bit what are the major filtering steps that determine whether a TF will be retained in the final result?
